package.json

```diff
@@ -59,6 +59,7 @@
     "@aws-sdk/client-redshift-data": "^3.382.0",
     "@aws-sdk/client-s3": "^3.304.0",
     "@aws-sdk/lib-dynamodb": "^3.386.0",
+    "@smithy/node-http-handler": "^2.1.5",
```

Comment:
why the new dep?

Reply:
to use the NodeHttpHandler for custom timeout
```
    this.client = new S3Client({
      requestHandler: new NodeHttpHandler({
        requestTimeout: 500,
      }),
    });
```